### PR TITLE
Custom/add quickstarter metadata

### PR DESCRIPTION
**Pull request by georgfedermann**

### CODE REVIEW

File: e2e-cypress/files/metadata.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+metadata:
+  name: Cypress
+  description: "Cypress is a JavaScript-based end-to-end testing framework that doesn't use Selenium at all. It is built on top of Mocha, which is again a feature-rich JavaScript test framework running on and in the browser, making asynchronous testing simple and fun. Technologies: Typescript"
```
Comment (metmajer) [inline e2e-cypress/files/metadata.yml]: @georgfedermann the offered technology here is Cypress, not TypeScript. Please consider putting that as a first item under Technologies. Cypress version used is 3.

File: fe-ionic/files/metadata.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+metadata:
+  name: Ionic
+  description: "Ionic Framework is the free, open source mobile UI toolkit for developing high-quality cross-platform apps for native iOS, Android, and the web—all from a single codebase. Technologies: Ionic 3.9.2"
```
Comment (metmajer) [inline fe-ionic/files/metadata.yml]: @georgfedermann can you double-check? I thought I had found a 4.3.2.

File: fe-react/files/metadata.yml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+---
+metadata:
+  name: React
+  description: "React is a JavaScript library for building user interfaces. It is maintained by Facebook and a community of individual developers and companies. Technologies: react unpinned"
```
Comment (metmajer) [inline fe-react/files/metadata.yml]: @georgfedermann can you double-check? I thought I had found a 16.4.

Comment (georgfedermann) [reply fe-react/files/metadata.yml]: I only found this in the Jenkinsfile:
`npm init react-app ${componentId}`
Thus thought, the react version was unpinned. Where did you find the v16.4?

File: docker-plain/files/metadata.yml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+---
+metadata:
+  name: plain docker alpine image
+  description: alpine latest
+  supplier: docker.com
```
[the remark this answers is not in the document]
Comment (metmajer) [reply docker-plain/files/metadata.yml]: @georgfedermann let me correct this. Alpine is currently our standard, but that should also be of no immediate interest to the end-users. Let's use the following for now:

```
metadata:
  name: Docker
  description: A plain component to dockerize your existing application.
  supplier: opendevstack.org
  version: 2.x
```

File: fe-react/files/metadata.yml (hunk 3 above)
Comment (metmajer) [reply fe-react/files/metadata.yml]: Hmm... `npm init react-app` requires the `create-react-app` [npm package](https://www.npmjs.com/package/create-react-app), but I cannot see where that is installed.

**Comment by metmajer** [comment]
Closes #75

**Pull request merged**